sdk/nestjs/.eslintrc.json

```diff
@@ -14,7 +14,7 @@
                 "@nx/dependency-checks": [
                     "error",
                     {
-                        "ignoredDependencies": []
+                        "ignoredDependencies": ["nestjs-cls"]
```

Comment:
is the linter complaining about this? what's it saying?

Reply:
it's complaining that the package is referenced in package.json but it's not used by the project